Marlin/Configuration_adv.h

```diff
@@ -3399,58 +3397,66 @@
+    
       /**
        * Scale the laser's power in proportion to the movement rate.
        *
        * - Sets the entry power proportional to the entry speed over the nominal speed.
        * - Ramps the power up every N steps to approximate the speed trapezoid.
        * - Due to the limited power resolution this is only approximate.
        */
-      #define LASER_POWER_INLINE_TRAPEZOID
+    //#define LASER_POWER_INLINE_TRAPEZOID
+    
 
-      /**
-       * Continuously calculate the current power (nominal_power * current_rate / nominal_rate).
-       * Required for accurate power with non-trapezoidal acceleration (e.g., S_CURVE_ACCELERATION).
-       * This is a costly calculation so this option is discouraged on 8-bit AVR boards.
-       *
-       * LASER_POWER_INLINE_TRAPEZOID_CONT_PER defines how many step cycles there are between power updates. If your
-       * board isn't able to generate steps fast enough (and you are using LASER_POWER_INLINE_TRAPEZOID_CONT), increase this.
-       * Note that when this is zero it means it occurs every cycle; 1 means a delay wait one cycle then run, etc.
-       */
-      //#define LASER_POWER_INLINE_TRAPEZOID_CONT
+    /**
+     * Continuously calculate the current power (nominal_power * current_rate / nominal_rate).
+     * Required for accurate power with non-trapezoidal acceleration (e.g., S_CURVE_ACCELERATION).
+     * This is a costly calculation so this option is discouraged on 8-bit AVR boards.
+     *
+     * LASER_POWER_INLINE_TRAPEZOID_CONT_PER defines how many step cycles there are between power updates. If your
+     * board isn't able to generate steps fast enough (and you are using LASER_POWER_INLINE_TRAPEZOID_CONT), increase this.
+     * Note that when this is zero it means it occurs every cycle; 1 means a delay wait one cycle then run, etc.
+     */
+    //#define LASER_POWER_INLINE_TRAPEZOID_CONT
 
-      /**
-       * Stepper iterations between power updates. Increase this value if the board
-       * can't keep up with the processing demands of LASER_POWER_INLINE_TRAPEZOID_CONT.
-       * Disable (or set to 0) to recalculate power on every stepper iteration.
-       */
-      //#define LASER_POWER_INLINE_TRAPEZOID_CONT_PER 10
+    /**
+     * Stepper iterations between power updates. Increase this value if the board
+     * can't keep up with the processing demands of LASER_POWER_INLINE_TRAPEZOID_CONT.
+     * Disable (or set to 0) to recalculate power on every stepper iteration.
+     */
+    //#define LASER_POWER_INLINE_TRAPEZOID_CONT_PER 10
 
-      /**
-       * Include laser power in G0/G1/G2/G3/G5 commands with the 'S' parameter
-       */
-      //#define LASER_MOVE_POWER
+      
+    /**
+     * Include laser power in G1/G2/G3/G5 commands with the 'S' parameter
+     */
+    //#define LASER_MOVE_POWER
 
-      #if ENABLED(LASER_MOVE_POWER)
+      // Discussion - G0 and G28 should never allow laser power! Travel and homming are not powered, so why have a feature for it?
```

Comment:
These options cause the laser to be TURNED OFF on a `G0` / `G28` instead of leaving the laser on. If the new code already automatically turns off the laser for `G0` / `G28` or other non-`G0`/`1`/`2`/`3`/`5` moves then these options may be removed.

Reply:
Agreed, LASER_MOVE_POWER is no longer required and we can remove it.